lib/cli/pack.ts

```diff
@@ -84,12 +92,6 @@ export const packHandler = async ({ handlerPath, outputFolder, publicFolder, sta
 				path: '**/*',
 				ignore: ['**/node_modules/**', '*.zip'],
 			},
-			{
-				// Ensure hidden files are included.
-				isGlob: true,
-				cwd: standaloneFolder,
-				path: '.*/**/*',
-			},
```

Comment:
ℹ️  The .next folder is in a subfolder in the case of monorepo. So I add the option "dot" to true of the archiver to include also the hidden files.